### PR TITLE
badges

### PR DESCRIPTION
**Pull request by samuray90**
** Late Pull Request** Easy way to build different shapes sizes and color for your badge (.badge)

### CODE REVIEW

File: pug-sass/src/index.pug [hunk 1]
```diff
@@ -9,3 +9,4 @@ html(lang="en")
     nav
       a(href="/marianzburlea") Marian Zburlea
       a(href="/bogdannegura") Bogdan Negura
+      a(href="/ionut") Ionut Daniel
```
Comment (marianzburlea) [inline pug-sass/src/index.pug:12]: is this your **github** id?

File: pug-sass/src/ionut/index.pug [hunk 2]
```diff
@@ -0,0 +1,23 @@
+html(lang="en")
+  head
+    meta(charset="UTF-8")
+    meta(name="viewport", content="width=device-width, initial-scale=1.0")
+    title PUG-sass Solutions with Marian Zburlea
+    link(rel="stylesheet", href="index.css")
+  body
+    h3 Badges with different colours, size and fills.
+    hr
+.column(span="row" height="10")
```
Comment (marianzburlea) [inline pug-sass/src/ionut/index.pug:10]: why is column starting at the same level as the **html** tag

File: pug-sass/src/ionut/index.pug [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    ul.child__list
+            li.badge.child__one Child One
```
Comment (marianzburlea) [inline pug-sass/src/ionut/index.pug:12]: crazy spacing

File: pug-sass/src/ionut/index.pug [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+            li.badge.child__two Child two
+            li.badge.child__three Child three
+            li.badge.child__four Child four
+            //- li.badge.child__five Child five
+            li.badge.child__six Child six
+            li.badge.child__ugly Child ugly
+            li.badge.child__smelly Smelly Child
+            //- li.badge.child__hoppy Child hoppy
+.column(span="row")
+  hr
```
Comment (marianzburlea) [inline pug-sass/src/ionut/index.pug:22]: you should put this inside the body

File: pug-sass/src/index.pug (hunk 1 above)
Comment (samuray90) [reply pug-sass/src/index.pug]: updated to samuray90

File: pug-sass/src/ionut/index.pug (hunk 3 above)
Comment (samuray90) [reply pug-sass/src/ionut/index.pug:12]: updated with the better spacing

File: pug-sass/src/ionut/index.pug (hunk 4 above)
Comment (samuray90) [reply pug-sass/src/ionut/index.pug:22]: all done

**Comment by marianzburlea** [comment]
bad name and you messed up the files at the root of the project.

**Pull request closed without merging**